config.py

```diff
@@ -0,0 +1,6 @@
+from json import loads
+from os import environ
+
+BOT_TOKEN = environ.get("BOT_TOKEN")
+PORT = int(environ.get("PORT", "5000"))
+GH_WEBHOOKS = loads(environ.get("GH_WEBHOOKS", "{}"))
```

Comment:
Is it possible to have heroku support without adding `config.py` to git?

Since this file is supposed to be ignored by git, for a normal deployment (run on a linux distro), it's better to manually write the dict directly in `config.py`.

Reply:
should / can I add one more environment variable like is_heroku, which will load sample_config.py, and rename the current config.py to sample_config.py?

Similar to the popular https://github.com/PaulSonOfLars/tgbot/blob/master/tg_bot/__init__.py#L21

Reply:
I just checked the default environment variable of Heroku dyno, including

```
declare -x DYNO="run.4152"
declare -x DYNO_RAM="512"
declare -x HOME="/app"
```

I think it's better to check `DYNO` variable in `config.py` (add it to git), if it's set then load configuration files from environment variable, else load from `config.json (ignore by .gitignore).

I'll do this to `heroku` branch and may merge it to default branch later.